src/Styleguide/Pages/Artwork/Sidebar/AuctionPartnerInfo.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react"
+import { Serif } from "@artsy/palette"
+import { Box } from "Styleguide/Elements/Box"
+
+export interface AuctionPartnerInfoProps {
+  artwork: {
+    readonly is_biddable: boolean
+    readonly partner: {
+      readonly __id: string
+      readonly name: string
+    }
+    readonly sale_artwork?: {
+      readonly estimate?: string
+    }
+    readonly sale?: {
+      readonly is_with_buyers_premium?: boolean
+    }
+  }
+}
+
+const AuctionPartnerInfoContainer = Box
```

Comment:
This is nice, I've started using this pattern too

Reply:
Yea... I think it increases readability in code but lame part that it does not seem to prepend this name when exploring things in inspector in the browser. Need to figure out if there is a way to force that. Makes debugging nicer:)